tests/Javascript/Components/Tables/RoleTable.spec.js

```diff
@@ -0,0 +1,63 @@
+import {beforeEach, jest, test} from "@jest/globals";
+
+jest.mock('laravel-jetstream')
+
+import {createLocalVue, shallowMount} from '@vue/test-utils'
+import {InertiaApp} from '@inertiajs/inertia-vue'
+import {InertiaForm} from 'laravel-jetstream'
+import RoleTable from '@src/Components/Tables/RoleTable'
+import {InertiaFormMock} from "@test/__mocks__/laravel-jetstream";
+
+let localVue;
+
+beforeEach(() => {
+  InertiaFormMock.error.mockClear()
+  InertiaFormMock.post.mockClear()
+
+  localVue = createLocalVue()
+  localVue.use(InertiaApp)
+  localVue.use(InertiaForm)
+
+});
+afterEach(() => {
+  localVue = null
+})
```

Comment:
random q,  do you think its possible to extract this somehow? I can see that the localVue setup is definitely something all tests need.

Reply:
No, local ur configuration Vaires from test to test, so it need to be defined every time.

Reply:
Makes sense but we do have repeated code in there. ex: all vue components should load inertia and form. would be nice to have a similar setup to PHPUnit tests where subclasses can replace the beforeEach() method should the need arise.